src/components/image-answer.tsx

```diff
@@ -115,11 +139,12 @@ export function ImageAnswer({ clue }: ImageAnswerProperties) {
               key={`${file.name}-${file.lastModified}`}

```

Comment:
This is basically the same as `getFileKey()`, can't they use the same method to keep things DRY?